src/hooks/use-authentication/index.ts

```diff
@@ -23,6 +30,9 @@ export const DEFAULT_PROVIDER_ID = ValidAuthProviderId.CloudIdp
 const useAuthentication = (
 	options: UseAuthenticationOptions = {}
 ): UseAuthenticationResult => {
+	const { flags } = useFlags()
```

Comment:
[question] is there a reason we're opting to use HappyKit here vs. the flags from config? Thinking specifically because you mentioned in the video that this should never be on for production environments, in which case a flag from config would be preferable as it allows the build to remove any code behind a conditional.

Reply:
Yep! I wanted the flag to be controllable during runtime rather than build time. I set up the production version of the flag to always be false whether it's turned on or off, but I think we can also put in some extra code that checks the current environment to make this more stable in the code.

![image](https://user-images.githubusercontent.com/43934258/201222015-f8c021c0-90c4-4e43-83b7-2908158a1355.png)

Reply:
Are we planning on extracting out the flag code after we feel this is stable?

Reply:
Yep!

Reply:
🗒️ PR has been updated and this convo is no longer relevant, so marking resolved.